libs/@local/hash-backend-utils/src/temporal-workflow-types.ts

```diff
@@ -0,0 +1,57 @@
+import { VersionedUrl } from "@blockprotocol/type-system";
+import { Issue, Team, User } from "@linear/sdk";
+import { Entity, EntityPropertiesObject } from "@local/hash-subgraph";
+
+export type PartialEntity = {
+  properties: Partial<Entity["properties"]>;
+  entityTypeId: VersionedUrl;
+};
+
+export type CreateHashIssueWorkflow = (params: {
+  payload: Issue;
+  ownedById: string;
+  actorId: string;
```

Comment:
I'm wondering why we're not using `OwnedById` and `ActorId` to type these? Is this to simplify calling them in temporal where they are typed as strings?

Reply:
No reason – done in https://github.com/hashintel/hash/pull/2834/commits/53c90c0af5ab9007c65cf8670f571585de51c695